# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1338935736 25200
#      Tue Jun 05 15:35:36 2012 -0700
# Branch gui
# Node ID 86f5cdf84fae0e739c8476809c05e4323c3cb1a1
# Parent  52640aa9c839576cdb0c6387938c95086103e50d
maint: Remove execute permission bits from png icons.

* artsbuilderexecute.png, configure.png, editcopy.png, editcut.png,
editpaste.png, filenew.png, fileopen.png, filesave.png, filesaveas.png,
redo.png, search.png, undo.png, up.png: Remove execute permission bit.

diff --git a/gui/src/icons/artsbuilderexecute.png b/gui/src/icons/artsbuilderexecute.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/configure.png b/gui/src/icons/configure.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/editcopy.png b/gui/src/icons/editcopy.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/editcut.png b/gui/src/icons/editcut.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/editpaste.png b/gui/src/icons/editpaste.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/filenew.png b/gui/src/icons/filenew.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/fileopen.png b/gui/src/icons/fileopen.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/filesave.png b/gui/src/icons/filesave.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/filesaveas.png b/gui/src/icons/filesaveas.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/redo.png b/gui/src/icons/redo.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/search.png b/gui/src/icons/search.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/undo.png b/gui/src/icons/undo.png
old mode 100755
new mode 100644
diff --git a/gui/src/icons/up.png b/gui/src/icons/up.png
old mode 100755
new mode 100644
